### PR TITLE
Updated README.md

### LINKED ISSUE #10: Create external and internal documentation
## Description
As a user, I would like to have an external (user interface/frontend) documentation to understand how the UI should be used. Add details about the implementation of the frontend with mock-up screens and explain the user interaction.

As a developer, I would like to have internal API documentation to understand API request and responses for the different end points. Add backend API documentation and explain how to use the exposed API.

As a project review and a developer, I would like to have a document on branching strategy and rules being followed.

All these should be documented in the wiki. README must have the links to this wiki and content of README can be moved to a "Home/About" page in Wiki.

## Acceptance Criteria
- [x] Add images for Edit Screen
- [x] Add images for User registration Screen
- [x] Add documentation for Edit screen
- [x] Add documentation for User registration Screen
- [x] Add images for Login Screen
- [x] Add images for Home Screen
- [x] Add image for navbar
- [x] Add documentation for Login screen
- [x] Add documentation for Home Screen
- [x] Add documentation for User table API end points
- [x] Add documentation for Post table API end points
- [x] Add documentation for branching strategy.
- [x] Add wiki links to README.
- [x] Move README content to a "Home/About" page in wiki.

### LINKED ISSUE #33: Setup Issue templates
## Description

As a developer, I need a way to automatically receive a standard format to create an user story or a bug.

I will be modifying this standard format with my own details (like description, acceptance criterias, steps to reproduce, etc...) and create an issue.

## Acceptance Criteria
- [x] Create an issue template for User story
- [x] Create an issue template for Bug
- [x] Create an issue template for Epic

### PR DESCRIPTION
**Pull request by shashank136**
## Acceptance Critera
- [x] Add wiki links to README.
- [x] Add Sprint1.md link to README. Refer #35 . 
- [x]  Move README content to a "Home/About" page in wiki.

### CODE REVIEW

**Comment by haxxorsid** [comment]
@shashank136 I have moved everything in README to Home page of wiki. You can clear everything in README, except group member names  and add all 4-5 Wiki page links.

Once, we have a Sprint1.md, we should link that as well from README.

**Comment by shashank136** [comment]
> @shashank136 I have moved everything in README to Home page of wiki. You can clear everything in README, except group member names and add all 4 Wiki page links (you left out Home page).
> 
> Once, we have a Sprint1.md, we should link that as well from README.

Updated the readme file to direct reader to Wiki pages

File: README.md [hunk 1]
```diff
@@ -7,46 +7,13 @@ A referral network to get referral from verified professionals.
...
-    - Common actions
--	Referrer
-    - Common actions
-    - View posts for referrals made for this user's company.
-    - Reveal candidate button on the post.
-    - Future **enhancement**, "I referred the candidate" button on the post, this will remove the post to avoid other existing employees again referring the user.
+- [API Reference](https://github.com/haxxorsid/referralboard/wiki/API-Reference)
+- [front-end Documentation](https://github.com/haxxorsid/referralboard/wiki/Frontend-Documentation)
 
+### How to raise an bug or feature request?
```
Comment (haxxorsid) [inline README.md]: "How to raise an bug or feature request?" is about creating an issue. Our branching strategy only talks about creating Pull requests and branches. So, you can write "Branching Strategy" or "Git Branching Strategy" instead.

File: README.md [hunk 2]
```diff
@@ -7,46 +7,13 @@ A referral network to get referral from verified professionals.
 - [Vineet Khadloya](https://github.com/Vineetkhadloya)
 - [Anjali Baheti](https://github.com/bahetianjali)
 
-## Technology
-- Go
-- React (Typescript)
-- Postgres 14.1.
+### About the project
```
Comment (haxxorsid) [inline README.md]: Need better heading.

Comment (shashank136) [reply README.md]: Suggestions are welcome 😅

File: README.md [hunk 3]
```diff
@@ -7,46 +7,13 @@ A referral network to get referral from verified professionals.
...
-    - Common actions
--	Referrer
-    - Common actions
-    - View posts for referrals made for this user's company.
-    - Reveal candidate button on the post.
-    - Future **enhancement**, "I referred the candidate" button on the post, this will remove the post to avoid other existing employees again referring the user.
+- [API Reference](https://github.com/haxxorsid/referralboard/wiki/API-Reference)
+- [front-end Documentation](https://github.com/haxxorsid/referralboard/wiki/Frontend-Documentation)
 
+### GIT Branching Strategy?
```
Comment (haxxorsid) [inline README.md]: remove question mark

File: README.md (hunk 2 above)
Comment (haxxorsid) [reply README.md:10]: Wiki or Getting Started or Documentation or Learn More.

**Review by haxxorsid — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (haxxorsid) [reply README.md]: Wiki or Getting Started or Documentation or Learn More or Index

**Review by bahetianjali — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -7,46 +7,12 @@ A referral network to get referral from verified professionals.
 - [Vineet Khadloya](https://github.com/Vineetkhadloya)
 - [Anjali Baheti](https://github.com/bahetianjali)
 
-## Technology
-- Go
-- React (Typescript)
-- Postgres 14.1.
+### Wiki
+- [Wiki](https://github.com/haxxorsid/referralboard/wiki)
```
Comment (haxxorsid) [inline README.md]: Line 11 About or About project or About the project?

**Review by haxxorsid — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -7,46 +7,12 @@ A referral network to get referral from verified professionals.
 - [Vineet Khadloya](https://github.com/Vineetkhadloya)
 - [Anjali Baheti](https://github.com/bahetianjali)
 
-## Technology
-- Go
-- React (Typescript)
-- Postgres 14.1.
+### Wiki
+- [About the project?](https://github.com/haxxorsid/referralboard/wiki)
```
Comment (haxxorsid) [inline README.md]: Drag Racing

File: README.md [hunk 6]
```diff
@@ -7,46 +7,12 @@ A referral network to get referral from verified professionals.
 - [Vineet Khadloya](https://github.com/Vineetkhadloya)
 - [Anjali Baheti](https://github.com/bahetianjali)
 
-## Technology
-- Go
-- React (Typescript)
-- Postgres 14.1.
+### Wiki
```
Comment (haxxorsid) [inline README.md]: ##Wiki not ###Wiki

**Review by haxxorsid — CHANGES_REQUESTED** [review]

**Review by haxxorsid — DISMISSED** [review]
Approved

**Review by haxxorsid — DISMISSED** [review]

**Review by Vineetkhadloya — DISMISSED** [review]
Good.

**Review by haxxorsid — DISMISSED** [review]

**Review by bahetianjali — DISMISSED** [review]

**Review by haxxorsid — DISMISSED** [review]

**Review by haxxorsid — APPROVED** [review]

**Review by bahetianjali — APPROVED** [review]

**Review by Vineetkhadloya — APPROVED** [review]
Good Job !!

**Pull request merged**